pincer/middleware/__init__.py

```diff
@@ -48,6 +48,7 @@ def get_middleware() -> Dict[str, Coro]:
                 "export"
             )()
         except AttributeError:
+            continue # TODO: Fix this. Always raises error because some modules are empty
```

Comment:
Can you add a debug log to let the users know what middlewares are missing for now? thx